<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>541</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="2">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="ClockRB">
      <property name="text">
       <string>Zegar</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="TimerRB">
      <property name="text">
       <string>Minutnik</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="RemainingRB">
      <property name="text">
       <string>Odlicz czas</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="AlarmRB">
      <property name="text">
       <string>Budzik</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="StopwatchRB">
      <property name="text">
       <string>Stoper</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="MinuteLE"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="AnulujB">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="StartB">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLabel" name="EkranLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="HourLE"/>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="MusicB">
      <property name="text">
       <string>ðŸŽµ</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="ResetB">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>â—¼</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
